### PR TITLE
Modify project sidebar

### PR DESCRIPTION
**Pull request by kaiqianyyang**
## Description

- change the view other projects url to http://localhost:3000/projects,
- delete contact button
- rename Team to Team&Alumni

## Updates

[Please enter a brief summary of updates to this PR during code review - if any]

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by kaiqianyyang** [comment]
[Modify project sidebar](https://trello.com/c/X4VHE9wX/82-modify-project-sidebar)

File: src/components/ProjectNavbar/ProjectNavbar.tsx [hunk 1]
```diff
@@ -139,7 +139,6 @@ export default function Sidebar(props: any) {
                         <Route exact path={`${props.match.url}/join`} render={() => <ProjectJoin project={props.currProject}/>}/>
                         <Route exact path={`${props.match.url}/resources`} render={() => <ProjectResources project={props.currProject}/>}/>
                         <Route exact path={`${props.match.url}/team`} render={() => <ProjectTeam project={props.currProject}/>}/>
-                        <Route exact path={`${props.match.url}/contact`} render={() => <ProjectContact project={props.currProject}/>}/>
```
[the remark this answers is not in the document]
Comment (svolterra) [reply src/components/ProjectNavbar/ProjectNavbar.tsx:142]: I was also thinking, if we no longer need a Contact page, would this file [here](https://github.com/UBC-VCL/VCL-content-platform/blob/main/src/pages/Project/ProjectContact.tsx) be necessary to keep? 🤔 Will follow-up and let you know! :)

**Review by svolterra — COMMENTED** [review]
Overall great work! Just a few minor suggestions here and there :) Thanks!

File: .eslintrc.yml [hunk 2]
```diff
@@ -14,5 +14,5 @@ parserOptions:
 plugins:
   - react
   - react-hooks
-  - '@typescript-eslint'
+  # - '@typescript-eslint'
```
[the remark this answers is not in the document]
Comment (kaiqianyyang) [reply .eslintrc.yml:17]: Well, this line was failing my local and two linting checks... 
I will add it back

File: src/components/ProjectNavbar/ProjectNavbar.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaiqianyyang) [reply src/components/ProjectNavbar/ProjectNavbar.tsx:142]: I will remove the import

File: src/components/ProjectNavbar/ProjectNavbar.tsx [hunk 3]
```diff
@@ -120,7 +120,7 @@ export default function Sidebar(props: any) {
                     ))}
                     <Box textAlign='center' padding='50px'>
                         <Button onClick={() => {
-                            window.location.pathname=ROUTES.PROJECT.OVERVIEW
+                            window.location.pathname="projects"
```
[the remark this answers is not in the document]
Comment (kaiqianyyang) [reply src/components/ProjectNavbar/ProjectNavbar.tsx:123]: Good point!

**Review by svolterra — APPROVED** [review]
Looks good! Thank you :) Please squash and merge when you get the chance 🥳

**Pull request merged**